lib/kennel.rb

```diff
@@ -54,48 +54,88 @@ class << self
   self.err = $stderr
 
   class Engine
-    attr_accessor :strict_imports
+    def initialize(
+      update_generated: true,
```

Comment:
```suggestion
      generate: true,
```

Reply:
Can do, in which case I'll also rename `def generated` (e.g. to `def parts`). It's confusing that `generated` is both an in-memory thing, and also an on-filesystem thing.